data/translations/en/homepage.json

```diff
@@ -22,7 +22,6 @@
     "process_url": "https://www.usa.gov/election"
   },
   "alert": {
-    "alert_msg": "Due to COVID-19, there may be changes to when, where and how you vote in your upcoming elections. Check with your state election site for more information.",
```

Comment:
In our last conversation with Laura about the data template she said she wanted to save this value just in case. We should keep it for english and spanish since we will be receiving translated content for this.